src/features/usage-consolidation-accordion/summary-table/column-properties.js

```diff
@@ -21,11 +27,20 @@ const DEFAULT_SUMMARY_TABLE_COLUMN_MAPPING = {
   [DEFAULT_SUMMARY_TABLE_COLUMNS.UC_ACTIONS]: null,
 };
 
-export const getSummaryTableColumnProperties = (intl, customProps = {}) => {
+const DEFAULT_FORMATTER = (currency) => ({
```

Comment:
Functions is not constants so it may be better to call it as `getDefaultSummaryTableFormatter`